ckanext/xloader/config_declaration.yaml

```diff
@@ -0,0 +1,116 @@
+version: 1
+groups:
+  - annotation: ckanext-xloader settings
+    options:
+      - key: ckanext.xloader.jobs_db.uri
+        example: sqlite:////tmp/xloader_jobs.db
```

Comment:
At the moment [it's a default value](https://github.com/ckan/ckanext-xloader/blob/9db532439dbb8b88f7c9d11f1e573ff6c164e3b9/ckanext/xloader/db.py#L40). Can we leave this default, as it shouldn't do any harm.